README.md

```diff
@@ -1,12 +1,26 @@
-# Git Repository Template
+# KMM Templates
 
-Project repository template to set up all public projects at [Nimble](https://nimblehq.co/)
+Kotlin Multiplatform project repository template to set up all public projects at [Nimble](https://nimblehq.co/)
```

Comment:
```suggestion
Kotlin Multiplatform project repository template to set up all projects at [Nimble](https://nimblehq.co/)
```

Reply:
Fixed in https://github.com/nimblehq/kmm-templates/pull/11/commits/5ad4a3ba6ceb824980cf27c7e013587df8caf097